src/style.css

```diff
@@ -1,3 +1,62 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+
+  --main-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  font-size: 12px;
+  padding: 0 50px;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.header__logolink {
+  height: 40px;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.nav__elem {
+  margin-left: 20px;
+}
+
+.nav__elem:first-child {
+  margin-left: 0;
+}
+
+.nav__link {
+  display: block;
+  line-height: 60px;
```

Comment:
set a correct figma line-height value. And set `padding` to make an element height you need